designer/server/src/main/scala/pl/touk/nussknacker/ui/api/StatisticsApiHttpService.scala

```diff
@@ -1,42 +1,37 @@
 package pl.touk.nussknacker.ui.api
 
-import com.typesafe.scalalogging.LazyLogging
 import pl.touk.nussknacker.ui.api.description.StatisticsApiEndpoints
-import pl.touk.nussknacker.ui.api.description.StatisticsApiEndpoints.Dtos.StatisticError.{DbError, InvalidURL}
+import pl.touk.nussknacker.ui.api.description.StatisticsApiEndpoints.Dtos.StatisticError.{
+  DbError => ApiDbError,
+  InvalidURL
+}
 import pl.touk.nussknacker.ui.api.description.StatisticsApiEndpoints.Dtos.{StatisticError, StatisticUrlResponseDto}
 import pl.touk.nussknacker.ui.security.api.AuthenticationResources
-import pl.touk.nussknacker.ui.statistics.UsageStatisticsReportsSettingsDeterminer
+import pl.touk.nussknacker.ui.statistics.{DbError, UrlError, UsageStatisticsReportsSettingsDeterminer}
 
-import java.sql.SQLException
 import scala.concurrent.ExecutionContext
 
 class StatisticsApiHttpService(
     authenticator: AuthenticationResources,
     determiner: UsageStatisticsReportsSettingsDeterminer
 )(implicit executionContext: ExecutionContext)
-    extends BaseHttpService(authenticator)
-    with LazyLogging {
+    extends BaseHttpService(authenticator) {
 
-  private val endpoints    = new StatisticsApiEndpoints(authenticator.authenticationMethod())
-  private val errorMessage = "Statistics URL construction failed"
+  private val endpoints = new StatisticsApiEndpoints(authenticator.authenticationMethod())
 
   expose {
     endpoints.statisticUsageEndpoint
       .serverSecurityLogic(authorizeKnownUser[StatisticError])
       .serverLogic { _ => _ =>
         determiner
           .determineStatisticsUrl()
```

Comment:
I'm not sure if `determine` is still the best choice.  WDYT about `prepare` instead?

Reply:
I agree